packages/bits/demo/src/components/demo/wizard-v2/wizard-custom-layout/wizard-custom.component.less

```diff
@@ -10,24 +10,32 @@
     .nui-wizard-step-header {
         border: @nui-line-default solid;
         border-radius: 5px;
-
         .setCssVariable(border-color, nui-color-line-default);
         .setCssVariable(background, nui-color-bg-content);
-
+        flex-direction: row-reverse;
+        justify-content: flex-end;
         padding: 0 8px;
     }
 
-    .nui-wizard-step-header-selected {
+    .nui-wizard-step-header--selected {
         .setCssVariable(background-color, nui-color-active);
 
         color: @nui-color-text-light;
         border-color: transparent;
 
-        svg path {
-            fill: @nui-color-text-inverse;
+        .nui-wizard-step-label{
+            .setCssVariable(color, nui-color-text-inverse)
```

Comment:
If we keep the color overrides, I think we should use nui-color-text-light

Reply:
That said, maybe we can explore a more explicit way to set the colors?

Reply:
Agree, that 'inverse' color looks strange. I also agree with finding a more explicit way of setting the color of icons. Setting colors sounds more like a new task to me, which we decided to postpone on, didn't we? We may also need input from UX here since the specs lack details about colors. We requested wizard specs to be updated, so hopefully, we'll get some details soon.

Reply:
sounds good. yeah, a separate ticket works. do you think we should we remove these styles in the meantime?

Reply:
we'll see what we can do with this and all the rest in the next sprint, I really doubt we'll be able to complete it in the scope of the current one.